Pilot/pilotCommands.py

```diff
@@ -917,3 +898,207 @@ def execute( self ):
     self.__startJobAgent()
 
     sys.exit( 0 )
+
+class MultiLaunchAgent( CommandBase ):
```

Comment:
What if you make this command an extension of the `LaunchAgent` command? This would say a lot of code repetition, I think